src/DeviceInterface/InterfacePrototype.js

```diff
@@ -576,6 +581,10 @@ class InterfacePrototype {
                 if (this.globalConfig.isDDGDomain && e.data.removeUserData) {
                     this.removeUserData()
                 }
+
+                if (this.globalConfig.isDDGDomain && e.data.closeEmailProtection) {
```

Comment:
Not a blocker, but I wonder whether we should rather set the listener only on our domain, as in moving the `this.globalConfig.isDDGDomain` check before we even set the listener.